snapcraft/internal/repo/_platform.py

```diff
@@ -26,6 +26,7 @@
     '"elementary"',
     'debian',
     'neon',
+    'LinuxMint',
```

Comment:
LinuxMint is spelled in different ways across releases AFAIR. Please look at https://github.com/zyga/os-release-zoo/tree/master/linuxmint

At one time it was indeed `LinuxMint` but I think they corrected that now.